libs/ui/packages/native/src/components/Layout/ScrollListContainer/index.tsx

```diff
@@ -17,21 +17,22 @@ type ScrollListContainerProps = BaseStyledProps &
  ** This Layout is a wrapper for FlatList that accepts
  ** complex onScroll callback for react-native-reanimated.
  */
-const ScrollListContainer = ({
-  children,
-  onScroll,
-  horizontal = false,
-  ...props
-}: ScrollListContainerProps): JSX.Element => {
-  return (
-    <AnimatedFlatList
-      data={[...React.Children.toArray(children)]}
-      renderItem={({ item }: { item: React.ReactNode }) => <View>{item}</View>}
-      onScroll={onScroll}
-      horizontal={horizontal}
-      {...props}
-    />
-  );
-};
+const ScrollListContainer = forwardRef(
```

Comment:
We should start to type using `React.FC<>` and others, instead of typing the params.
```suggestion
const ScrollListContainer = forwardRef<HTMLInputElement, ScrollListContainerProps>(
```

Some examples with `React.FC<Props>`:
- https://github.com/LedgerHQ/ledger-live/pull/91/files#diff-7b7308896d5f33e74ff1fec9e6279a95f02fb5f41f6d430802105e50cf3d904dR27
- https://github.com/LedgerHQ/ledger-live/pull/91/files#diff-7b7308896d5f33e74ff1fec9e6279a95f02fb5f41f6d430802105e50cf3d904dR27

Reply:
thanks for the review & suggestion 👍